test/common/websocket/codec_test.cc

```diff
@@ -23,22 +95,15 @@ namespace {
 // opcode - 0x1 - text data
 // mask bit - 0
 // length - 5 bytes 0000101
-TEST(WebSocketCodecTest, decodeSingleUnmaskedFrame) {
+TEST(WebSocketCodecTest, DecodeSingleUnmaskedFrame) {
   Buffer::OwnedImpl buffer;
   Buffer::addSeq(buffer, {0x81, 0x05, 0x48, 0x65, 0x6c, 0x6c, 0x6f});
 
   Decoder decoder;
   absl::optional<std::vector<Frame>> frames = decoder.decode(buffer);
 
-  ASSERT_TRUE(frames.has_value());
-  EXPECT_EQ(false, frames.value()[0].masking_key_.has_value());
-  EXPECT_EQ(1, frames.value().size());
-  EXPECT_EQ(5, frames.value()[0].payload_length_);
-
-  std::string text_payload;
-  text_payload.resize(5);
-  (*(frames.value()[0].payload_)).copyOut(0, 5, text_payload.data());
-  EXPECT_EQ("Hello", text_payload);
+  EXPECT_TRUE(areFramesEqual(frames.value()[0], {true, kFrameOpcodeText, absl::nullopt, 5,
+                                                 std::make_unique<Buffer::OwnedImpl>("Hello", 5)}));
```

Comment:
nit: It's slightly annoying to have to specify both the data and the length in all of these cases. How bout adding a helper method like:

```
BufferPtr makeBuffer(absl::string_view data) {
  return std::make_unique<Buffer::OwnedImpl>(data.data(), data.length());
}
```

And then using it throughout this test..